openfisca_us/variables/usda/usda.py

```diff
@@ -0,0 +1,20 @@
+from openfisca_core.model_api import *
+from openfisca_us.entities import *
+from openfisca_us.tools.general import *
```

Comment:
needs `import numpy as np`

Reply:
Seems to be working without this - I believe it's already imported elsewhere

Reply:
Yep, the wildcard import from `openfisca_us.tools.general` does it